docs/pages/dsFPD/ds-front-panel-display_L3_TestProcedure.md

```diff
@@ -101,15 +101,7 @@ dsFPD:
   assets:
     device:
       defaults: &defaults
-        artifacts:
-          - "<PATH>/bin/hal_test"
-          - "<PATH>/bin/libut_control.so"
-          - "<PATH>/profiles/source/Source_FPD.yaml"
-          - "<PATH>/profiles/sink/Sink_FPD.yaml"
```

Comment:
The description for dsFPD_L3_Runall.py has not been implemented

Reply:
Added section for dsFPD_L3_Runall.py